### PR TITLE
feat(helm): add helm update command

### PR DESCRIPTION
**Pull request by michelleN**
see #640

### CODE REVIEW

File: cmd/helm/update.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+	var cacheFile *os.File
+	var r repo.RepoFile
+
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+
+	if err := yaml.Unmarshal(b, &r); err != nil {
```
Comment (vaikas-google) [inline cmd/helm/update.go]: I assume the unmarshaling is done to validate that the file at least looks like what we'd expect. Do we need to do any further validation of the file that it indeed has all the required fields, etc. and have sane values? If so, then we should probably hoist this to a method. But perhaps the unmarshaling is enough?

**Comment by vaikas-google** [comment]
LGTM barring the discussion about moving to under 'repo' subcommand.

File: cmd/helm/update.go (hunk 1 above)
Comment (michelleN) [inline cmd/helm/update.go]: Yep, you are correct. It's just to validate that the file looks correct with the correct fields.

**Comment by michelleN** [comment]
@vaikas-google and I decided it was fine to go with `helm update` for now, so merged.

**Pull request merged**